packages/samples/mvp-dapp/.prettierrc

```diff
@@ -0,0 +1,6 @@
+{
+  "printWidth": 120,
+  "trailingComma": "all",
+  "singleQuote": true,
+  "arrowParens": "always"
```

Comment:
A point was neglected before that it's better to keep the rule consistent: https://github.com/ckb-js/kuai/blob/develop/.prettierrc#L6